src/db.rs

```diff
@@ -583,10 +614,9 @@ mod tests {
         let pkcs8_dummy_bytes = [0xa5u8; 512];
 
         db.insert_report_keypair(&tid, &pkcs8_dummy_bytes, None)
-            .await
-            .unwrap();
+            .await?;
```

Comment:
I think it's fine to `unwrap` within tests btw.

Reply:
Ok, thanks.  I will keep this in mind for future updates.  Because we are interested in the result state, I left the await?.